src/components/pet/pet.component.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, Fragment } from 'react';
+
+import { ObservationsComponent } from './components/observations/observations.component';
+import { AdditionalComponent } from './components/additional/additional.component';
+import { CharacteristicsComponent } from './components/characteristics/characteristics.component';
+import { Logo } from './components/logo/logo.component';
+import './pet.css';
+
+interface PetProps {
+  match: {
+    params: {
+      id: number;
+    };
+  };
+}
+export const Pet = (props: PetProps): React.ReactElement => {
+  const petId: number = props.match.params.id;
+  const [imgUrl, setImgUrl] = React.useState<string>('');
+  const [breed, setBreed] = React.useState<string>('');
+  const [characteristics, setCharacteristics] = React.useState();
+  const [observations, setObservations] = React.useState();
+  const [additionalInfo, setAdditionalInfo] = React.useState();
+  const [loading, setLoading] = React.useState<boolean>(true);
```

Comment:
Why don't you destructure useState into import like useEffect. 
So you avoid repeating React. in each line

Reply:
Good point, thanks. Corrected